File: EmployeeManagement/WebApplication1/Enums/Gender.cs
using System.ComponentModel;

namespace EmployeeManagement.Enums
{
    public enum Gender
    {
        [Description("Woman")]
        Woman = 1,
        [Description("Man")]
        Man = 2,
        [Description("Transgender")]
        Transgender = 3,
        [Description("Non binary")]
        NonBinary = 4,
        [Description("Prefer not to respond")]
        PreferNotToRespond = 5,
    }
}

File: employee-management/src/utils/gender.ts
export const Gender = (value : number) => {
  switch (value) {
    case 1:
      return "Woman";
    case 2:
      return "Man";
    case 3:
      return "Transgender";
    case 4:
      return "Non binary";
    case 5:
      return "Prefer not to respond";
    default:
      return "Invalid gender";
  }
}

